src/components/Login.jsx

```diff
@@ -123,7 +188,7 @@ function Login({ setCredentials }) {
         );
       } else if (!result?.access_token) {
         window.alert(
-          "Login failed.\n\nThe response from the server was not understood.  Please reload or try again later"
+          "Login failed.\n\nThe response from the server was not understood.  Please reload or try again later."
```

Comment:
Consider moving the alert messages to constants to avoid magic strings and make the code more maintainable.
For example:

const INVALID_EMAIL_MSG = "That's not a valid e-mail address!  Who're you trying to kid?";

Reply:
Actually, these aren't [magic strings](https://en.wikipedia.org/wiki/Magic_string); [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) however, are indeed to be avoided and shunned.

Moving these strings to constants would actually increase the code's complexity without improving maintainability much (if at all).  The only use case that I can think of for doing this is internationalization, but that's not one of the project's goals.

*These alerts should be converted to toasts in the near future.  We can revisit the idea when this happens.*